IdentityProxyXHR.js

```diff
@@ -18,6 +18,7 @@
                 if (typeof isAsync !== "undefined" && !isAsync) {
                     throw "Synchronous XHR requests not supported";
                 }
+                _this.setProxyCoreByUrl(url);
```

Comment:
So the theory here is that the specific set of core details used by the proxy is swapped around as each request is processed? 

BTW it seems that we are missing equivalent changes to the IdentityProxyServiceWorker implementation